api/v1/openapi.yaml

```diff
@@ -3506,6 +3506,15 @@ definitions:
         type: array
         items:
           "$ref": "#/definitions/BgpPeerFamilies"
+      connect-retry-time:
```

Comment:
I'd prefer to have units in the field name (like `connect-retry-time-second`). So that users don't have to look into docs.